lib/std/Progress.zig

```diff
@@ -419,6 +419,17 @@ pub fn start(options: Options) Node {
                 };
             }
 
+            std.io.tty.setAbortSignalHandler(struct {
```

Comment:
One problem might be that this is a global handler. Once you use `std.Progress` in your program, either your program can use this to handle Ctrl+C if the handler is registered after `start` is called or `std.Progress` can use it.

If this is too much of a problem maybe adding something like `abort_signal_handlers: []const *const fn () void` to `std.Options` could be a solution?